.gitignore

```diff
@@ -3,3 +3,4 @@
 Gemfile.lock
 pkg/*
 .DS_Store
+.idea/
```

Comment:
It'd be better to create your own global gitignore (can search online how) to ignore local dev things like this. Otherwise you'll need to add this to every project you work on.